pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = ["setuptools", "wheel", "versioneer"]
```

Comment:
Mostly a todo for myself - I'm not sure you need to/should include versioneer in the build requirements if versioneer is already vendored.

Maybe as an alternative to versioneer - we've been moving a lot of stuff to versioningit instead (see https://github.com/MDAnalysis/PathSimAnalysis/pull/12 for an example). It gets rid of some messy toml dependencies whilst also avoiding vendoring things. Note: the real reason we're moving away from versioneer is that it hasn't been as consistently maintained lately, you could stay with versioneer for now without issues though.

Reply:
Ok I've worked out what's going wrong with the whole `versioneer` needing to be installed thing.

Turns out that it needs additional stuff for the vendored file to be picked up: https://github.com/python-versioneer/python-versioneer/issues/249

The diff would need to look something like this:

```
diff --git a/pyproject.toml b/pyproject.toml
index d1ed562..6249d7b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "versioneer"]
+requires = ["setuptools", "wheel", 'tomli; python_version < "3.11"']
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -62,4 +62,4 @@ include = ["membrane_curvature*"]
 
 [project.urls]
 Documentation = "https://membrane-curvature.readthedocs.io/en/latest/"
-Changelog = "https://github.com/MDAnalysis/membrane-curvature/blob/main/CHANGELOG.rst"
\ No newline at end of file
+Changelog = "https://github.com/MDAnalysis/membrane-curvature/blob/main/CHANGELOG.rst"
diff --git a/setup.py b/setup.py
index e4e8d2e..0880925 100644
--- a/setup.py
+++ b/setup.py
@@ -2,8 +2,12 @@
 MembraneCurvature MDAkit
 A tool to calculate Membrane Curvature
 """
+import os
 import sys
 from setuptools import setup, find_packages
+
+sys.path.append(os.path.dirname(__file__))
+
 import versioneer
 
 short_description = __doc__.split("\n")
```

Reply:
@ojeda-e at this point I would suggest just switching over to versioningit, it'll make life a lot easier.

Reply:
I've opened a PR into this PR with the necessary changes.

Reply:
Thanks @IAlibay!